djconnectwise/models.py

```diff
@@ -1012,6 +1012,26 @@ class Meta:
     def __str__(self):
         return self.name or ''
 
+    def update_cw(self):
+        """
+        Send project status updates to ConnectWise.
+        """
+        api_client = api.ProjectAPIClient()
+
+        return api_client.update_project(self.id, self.status)
+
+    def save(self, *args, **kwargs):
```

Comment:
Just wondering - will we need this? I could very well be missing something here, but it seems to me that most of the time we'd want to send an update to CW first and then save to our DB (provided the update to CW was succesful) not the other way around.

As for as I can tell the only time we use the 'update_cw' kwarg is in `service_close.py`: 29.